core-services/ci-secret-generator/_config.yaml

```diff
@@ -38,8 +38,8 @@
 - item_name: build_farm
   attachments:
   - cmd: |-
-      oc --context $(cluster) get secrets --namespace ci -o json | jq '.items[] | select(.type=="kubernetes.io/dockercfg") | select(.metadata.annotations["kubernetes.io/service-account.name"]=="build01") | .data[".dockercfg"]' --raw-output | base64 --decode | jq '.["image-registry.openshift-image-registry.svc:5000"].auth' --raw-output | tr -d '\n'
-    name: build01_$(cluster)_reg_auth_value.txt
+      oc --context $(cluster) get secrets --namespace ci -o json | jq '.items[] | select(.type=="kubernetes.io/dockercfg") | select(.metadata.annotations["kubernetes.io/service-account.name"]=="image-puller") | .data[".dockercfg"]' --raw-output | base64 --decode | jq '.["image-registry.openshift-image-registry.svc:5000"].auth' --raw-output | tr -d '\n'
+    name: image-puller_$(cluster)_reg_auth_value.txt
```

Comment:
While we're at it ... maybe `token.image-puller.$(cluster).txt` to match the other values we have in here?

Reply:
Updated.
I do not see other "token" in this file though.